core/sqf/sql/scripts/sqregen

```diff
@@ -41,8 +41,15 @@ function Usage {
 
 function RegenConfig {
     echo "Regenerating Trafodion configuration: $SQCONFIG_FILE"
+    cd $TRAF_HOME/sql/scripts
     ./regensq.pl $SQCONFIG_FILE
-    exit $?;
+    if [ $? != 0 ]; then
+        echo "Failed to run regensq.pl."
```

Comment:
The error message is incorrect. We did run regensq.pl. The problem is that regensq.pl failed.